ClubDay2022/Views/ContentView.swift

```diff
@@ -79,10 +79,10 @@ struct ContentView: View {
                 }
 
             } label: {
-                Text(model.isOn ? "ON" : "OFF")
+                Image(systemName: model.isOn ? "pause" : "play")
```

Comment:
`pause.fill` might be better

<img width="270" alt="Screen Shot 2022-10-20 at 8 30 00 PM" src="https://user-images.githubusercontent.com/49819455/197105102-e8bd8a2d-953b-4f4e-8144-f8e042c84c4d.png">

Reply:
Will update in an hour (MIDNIGHTS!)